tests/pipeline/test_engine.py

```diff
@@ -112,6 +111,16 @@ def compute(self, today, assets, out, open):
         out[:] = open
 
 
+class MultipleOutputs(CustomFactor):
+    window_length = 1
+    inputs = [USEquityPricing.open, USEquityPricing.close]
+    outputs = ['open', 'close']
```

Comment:
We probably want a test that exercises the behavior of passing `outputs` dynamically to the constructor.